deploy/service_account.yaml

```diff
@@ -6,3 +6,7 @@ metadata:
   labels:
     product: ibm-block-csi-driver
     csi: ibm
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+    release: ibm-block-csi-operator
```

Comment:
shouldn't this be `release: v1.4.0`?

Reply:
changed

Reply:
did you re-test the value? if so, could you do the same with all changes of this PR?

Reply:
now I see it, thanks